doc/nrf/protocols/matter/getting_started/transmission_power.rst

```diff
@@ -0,0 +1,154 @@
+.. _ug_matter_gs_transmission_power:
+
+Configuring transmission power
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can configure the radio transmitter power (TX power) of your device to increase or decrease the wireless communication range of your device.
+The radio transmitter power configuration must be modified separately for every wireless technology used in the Matter applications.
+
+TX power settings can have a significant impact on the device's power consumption when :ref:`ug_matter_device_low_power_configuration`.
+
+Default TX power
+****************
+
+The following table lists the default TX power values.
```

Comment:
Could we create similar entries for Bluetooth LE and Wi-Fi? It looks like quite generic section, however it describes only Thread and uses Thread-specific names like SED, MTD, FTD.

Reply:
Ok, it looks like for Wi-Fi it's not a constant value and it depends on several things like frequency band or type of modulation. Nevertheless we could refer to this page: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf7002%2Fchapters%2Felspec%2Fdoc%2Felectrical_specification.html, Table 3.

Reply:
For each sample, the default BLE TX power is set to 0 dBm ``CONFIG_BT_CTLR_TX_PWR_0``. There are some differences between the maximum possible values depending on a board:

**For nrf52840dk_52840:**
-  max BLE power is +8 dBm -> ``CONFIG_BT_CTLR_TX_PWR_PLUS_8``
- min BLE power is -40 dBm -> ``CONFIG_BT_CTLR_TX_PWR_MINUS_40``
- 
**For nrf5340dk_nrf5340 & nrf7002dk_nrf5340**:
-  max BLE power is +3 dBm -> ``CONFIG_BT_CTLR_TX_PWR_PLUS_3``
- min BLE power is -40 dBm -> ``CONFIG_BT_CTLR_TX_PWR_MINUS_40``
 
Maybe it is worth adding to the table or somewhere else.